### PR TITLE
More scratch

### PR DESCRIPTION
**Pull request by SamanthaDawsonBanks**
General workspace clean up

### CODE REVIEW

**Comment by SamanthaDawsonBanks** [comment]
BTW this one has some directory changes so you might need to clean and re import the projects after this has merged, so I will hold off of this one till we are next in person

**Review by RyanShoobert — APPROVED** [review]
Some nice changes here, approved. The addition of a C scratchpad will be helpful as I continue to develop the coms stuff.

**Review by HarryWJackson1996 — APPROVED** [review]
The cleanup looks fine, and as Ryan said a C scratchpad will be useful moving forward with the project, plus the removal of implied names is neater.

**Review by stevempope — APPROVED** [review]

**Pull request merged**
